linker issue (cannot produce executable)

I'm currently trying to run the game on macOS, but theres an issue producing the executable along with a bunch of warnings.  `make` fails at 91% with this error:

>[ 91%] Linking CXX executable ../bin/usc-game
Undefined symbols for architecture x86_64:
  "GameConfig::VERSION", referenced from:
      Application::m_LoadConfig() in Application.cpp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [bin/usc-game] Error 1
make[1]: *** [Main/CMakeFiles/usc-game.dir/all] Error 2
make: *** [all] Error 2

I've ran `git submodule update --init --recursive`, and I've installed all the deliverables.

I've also tried macOS 10.12 but no luck.

Desktop
 - OS: Mac 10.11.6
 - GPU: Intel HD Graphics 4000
 - CPU: Intel Core i7
 - Version: 0.4.0

I've also attached the log for the warnings

thanks in advance

[linker error (USC).log](https://github.com/Drewol/unnamed-sdvx-clone/files/5856094/linker.error.USC.log)

--- a/Main/include/GameConfig.hpp
+++ b/Main/include/GameConfig.hpp
@@ -213,10 +213,7 @@ public:
 	GameConfig();
 	void SetKeyBinding(GameConfigKeys key, Key value);
 
-
-	// When this should change, the UpdateVersion MUST be updated to update the old config files.
-	// If there's no need to update the UpdateVersion, there's no need to touch this too.
-	constexpr static int32 VERSION = 1;
+	static int32 VERSION;
 
 	// Update the version of the config file to VERSION.
 	void UpdateVersion();

--- a/Main/src/GameConfig.cpp
+++ b/Main/src/GameConfig.cpp
@@ -4,6 +4,10 @@
 #include "Shared/Log.hpp"
 #include "HitStat.hpp"
 
+// When this should change, the UpdateVersion MUST be updated to update the old config files.
+// If there's no need to update the UpdateVersion, there's no need to touch this too.
+int32 GameConfig::VERSION = 1;
+
 inline static void ConvertKeyCodeToScanCode(GameConfig& config, std::vector<GameConfigKeys> keys)
 {
 	// To use SDL_GetScancodeFromKey, SDL must be initialized before.

